### PR TITLE
Move yield source data here

### PR DESCRIPTION
**Pull request by chuckbergeron**

### CODE REVIEW

File: src/__tests__/yieldSources.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  it('throws with bad params', () => {
+    expect(() => {
+      getKnownYieldSourceContract()
+    }).toThrow()
+  })
+
+  it('throws with bad eth address', () => {
+    expect(() => {
+      const knownContract = getKnownYieldSourceContract(1, '0xface')
```
Comment (ddesrosier) [inline src/__tests__/yieldSources.test.ts:69]: Hahaha

File: src/__tests__/yieldSources.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    const knownContract = getKnownYieldSourceContract(
+      1,
+      '0x6b175474e89094c44da98b954eedeac495271d0f'
+    )
+    expect(knownContract).toEqual(undefined)
+  })
+
+  it('throws with bad params', () => {
+    expect(() => {
+      getKnownYieldSourceContract()
```
Comment (ddesrosier) [inline src/__tests__/yieldSources.test.ts:63]: The tests are failing for me here:

```
src/__tests__/yieldSources.test.ts:63:7 - error TS2554: Expected 2 arguments, but got 0.

    63       getKnownYieldSourceContract()
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

      src/yieldSources.ts:21:3
        21   chainId: number,
             ~~~~~~~~~~~~~~~
        An argument for 'chainId' was not provided.
```

**Review by ddesrosier — CHANGES_REQUESTED** [review]
Awesome! Thanks for setting up jest. One test is broken but the rest looks good

File: src/__tests__/yieldSources.test.ts (hunk 2 above)
Comment (chuckbergeron) [reply src/__tests__/yieldSources.test.ts:63]: oh yah, the TS strictness issue. I thought I commented that test out? Maybe forgot to push.

I looked it up this morn and I think I can pass `(chainId: any, address: any)` for undefined values.

**Pull request merged**